web_external/Studies/studyPage.styl

```diff
@@ -9,6 +9,8 @@
 
 .isic-study-users-content
   ul
+    list-style none
```

Comment:
In my own subjective opinion, this list is still pretty ugly, but re-adding the bullets doesn't seem to help. I'm fine with leaving it alone for now, but any future improvements are welcome.